fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailActivity.java

```diff
@@ -256,6 +256,9 @@ private void setupIdentityFragment() {
   }
 
   private void setupSocialMediaActivity() {
+    final String LAO_ID = "LAO_ID";
+    final String LAO_NAME = "LAO_NAME";
+    final String OPENED_FROM = "OPENED_FROM";
```

Comment:
Is this is related to the SocialMediaActivity, I would have placed the constants as public static final String inside that class.